core/models.py

```diff
@@ -58,26 +57,26 @@ class Question(models.Model):
     question_text = RichTextUploadingField()
     choice1 = RichTextUploadingField()
     choice2 = RichTextUploadingField()
-    choice3= RichTextUploadingField()
+    choice3 = RichTextUploadingField()
     choice4 = RichTextUploadingField()
     correct_choice = models.PositiveIntegerField(blank=False)
     negative = models.BooleanField(default=False)
     negative_marks = models.IntegerField(null=True, blank=True)
     marks = models.IntegerField(null=True, default=1)
 
-
     def __str__(self):
         return self.question_text
 
 
 class Candidate(models.Model):
     name = models.CharField(max_length=100, blank=False)
-    # std_no_regex = RegexValidator(regex=r"^\d{7}$")
-    std_no = models.CharField(blank=True, max_length=7, null=True)
-    university_roll_no = models.CharField(max_length=10, unique=True)
+    std_no_regex = RegexValidator(regex=r"^\d{7}$", message="Invalid Student Number", code="400")
```

Comment:
"Redundant work" phone number, student number, and university roll number validation already present in forms.py so no need of it here